opencog/nlp/learn/run/mst-one.sh

```diff
@@ -33,7 +34,7 @@ observe="observe-mst"
 # 	exit 1
 # fi
 # Alternate cogserver test: use netcat to ping it.
-haveping=`echo foo | nc -N $coghost $cogport`
+haveping=`echo foo | nc $coghost $cogport`
```

Comment:
Some versions of nc have to have the -N flag, and without it, nc will hang forever and do nothing. Other versions of nc don't need the -N flag.   Unfortunately, nc does not print what version it is, so I don't know how to tell when -N is needed, or not.